src/transport/raw/tests/TestPeerAddress.cpp

```diff
@@ -62,18 +67,76 @@ void TestPeerAddressMulticast(nlTestSuite * inSuite, void * inContext)
     NL_TEST_ASSERT(inSuite, !memcmp(expected, result, NL_INET_IPV6_ADDR_LEN_IN_BYTES));
 }
 
+void TestToString(nlTestSuite * inSuite, void * inContext)
+{
+    char buff[PeerAddress::kMaxToStringSize];
+    IPAddress ip;
+    {
+        IPAddress::FromString("::1", ip);
+        PeerAddress::UDP(ip, 1122).ToString(buff);
+
+        NL_TEST_ASSERT(inSuite, !strcmp(buff, "UDP:[::1]:1122"));
+    }
+
+    {
+        IPAddress::FromString("::1", ip);
+        PeerAddress::TCP(ip, 1122).ToString(buff);
+
+        NL_TEST_ASSERT(inSuite, !strcmp(buff, "TCP:[::1]:1122"));
+    }
+
+    {
+        PeerAddress::BLE().ToString(buff);
+        NL_TEST_ASSERT(inSuite, !strcmp(buff, "BLE"));
+    }
+
+    {
+        IPAddress::FromString("1223::3456:789a", ip);
+        PeerAddress::UDP(ip, 8080).ToString(buff);
+
+        NL_TEST_ASSERT(inSuite, !strcmp(buff, "UDP:[1223::3456:789a]:8080"));
+    }
+
+    {
+
+        PeerAddress udp = PeerAddress(Transport::Type::kUdp);
+        udp.SetPort(5840);
```

Comment:
Suggest a less confusing port like 23456. `5840` is close to default CHIP_PORT

Reply:
Does it matter?